ui/app/AppLayouts/Chat/components/MessageContextMenu.qml

```diff
@@ -186,34 +186,31 @@ PopupMenu {
         }
     }
 
-    Action {
+    StatusMenuItem {
         id: copyAction
-        enabled: !isProfile && !emojiOnly
+        enabled: false
```

Comment:
If this is always disabled, why keeping it?

Reply:
So what I go tot know is that this feature is present , just not supported yet. SO we may anyway have to bring this back in the future.

Reply:
TBH I am not sure :) I can see also that you put enabled: false in front of edit message which is I am almost sure supported. Maybe @richard-ramos @PascalPrecht knows more?

Reply:
https://www.figma.com/file/Mr3rqxxgKJ2zMQ06UAKiWL/%F0%9F%92%AC-Chat%E2%8E%9CDesktop?node-id=5395%3A43297 This is the design for  now. But @simonricoo mentioned copy link will come up in the future. Maybe I can remove the rest. What do you think @PascalPrecht

Reply:
I this context menu is being used also when you right click on a message, this might be why on this specific design, this action is not visible

Reply:
Yes my bad I didn't mention that. For Profile only two options exist for profile -> https://www.figma.com/file/Mr3rqxxgKJ2zMQ06UAKiWL/%F0%9F%92%AC-Chat%E2%8E%9CDesktop?node-id=749%3A1441

Reply:
Maybe just comment? Or maybe we have some issue for it?

Reply:
I am okay with that too. :)

Reply:
I just realised I did not take of the right click options correctly. Need to rework. In the mean time will comment "copy link" code. One thing that remains disabled is jumpTo, will just remove that.

Reply:
Have done it. Please check @borismelnik @alaibe  :)